numpy/core/code_generators/ufunc_docstrings.py

```diff
@@ -2474,6 +2475,10 @@ def add_newdoc(place, name, doc):
     -----
     For integer input the return values are floats.
 
+    See Also
+    --------
+    divmod : Simultaneous floor division and remainder.
```

Comment:
I was envisaging likening `divmod(x, 1)` to `modf` here, in the same way we do the reverse in `divmod`